pajbot/apiwrappers/twitch/helix.py

```diff
@@ -602,3 +602,17 @@ def get_channel_emotes(
             expiry=60 * 60,
             force_fetch=force_fetch,
         )
+
+    def send_chat_announcement(self, bot_id: str, body: Dict[str, str], authorization, channel_id: str = None):
+        if not body:
+            log.error("Invalid call to send_chat_announcement, missing query parameter. message must be specified")
+            return False
+
+        response = self.post(
+            "/chat/announcements",
+            {"broadcaster_id": channel_id, "moderator_id": bot_id},
```

Comment:
```suggestion
            {"broadcaster_id": channel_id, "moderator_id": sender_id},
```

Reply:
I'm not sure if I agree with this change, I intentionally called it `bot_id` to make it as simple and straightforward as possible.